pkg/backup/item_collector.go

```diff
@@ -741,6 +741,19 @@ func (r *itemCollector) collectNamespaces(
 		return nil, errors.WithStack(err)
 	}
 
+	for _, includedNSName := range r.backupRequest.Backup.Spec.IncludedNamespaces {
```

Comment:
Since "*" is a valid entry in `IncludedNamespaces`, we should not look for the literal namespace "*" if this is the value of `includedNSName`.

We probably want something like:
```
if includedNSName == "*" {
    continue
}
```

Reply:
Thanks. Modified.